kong/tools/wrpc/init.lua

```diff
@@ -170,382 +71,93 @@ function wrpc_peer:receive()
     end
 
     if typ == "close" then
-      kong.log.notice("Received WebSocket \"close\" frame from peer: ", err, ": ", data)
+      ngx_log(NOTICE, "Received WebSocket \"close\" frame from peer: ", err, ": ", data)
       return self:close()
     end
   end
 end
 
---- RPC call.
---- returns the call sequence number, doesn't wait for response.
-function wrpc_peer:call(name, ...)
-  local rpc, payloads = assert(self.service:encode_args(name, ...))
-  return self:send_encoded_call(rpc, payloads)
-end
```

Comment:
make this single-argument too (if `:encode_args()` no longer supports multiple args)

Reply:
I was planning to do this at another PR. But I think it's OK to include this.